modules/tensor_mechanics/test/tests/anisotropic_elastoplasticity/hoop_strain_comparison_coarse.i

```diff
@@ -0,0 +1,278 @@
+  [stress_yy]
+    type = ADRankTwoAux
+    rank_two_tensor = stress
+    variable = stress_yy
+    index_i = 1
+    index_j = 1
+  []
+[]
+
+[Functions]
+  [push]
+    type = PiecewiseLinear
+    x = '0 1e2'
+    y = '0 200e6'
+  []
+  [swelling_func]
+    type = ParsedFunction
+    expression = 0
+  []
+[]
+
+[Modules/TensorMechanics/Master]
+  [all]
+    strain = FINITE
+    generate_output = 'elastic_strain_zz elastic_strain_xx elastic_strain_yy stress_xx stress_yy stress_zz strain_zz plastic_strain_zz plastic_strain_xx plastic_strain_yy hoop_stress hoop_strain'
+    use_automatic_differentiation = true
+    add_variables = true
+    cylindrical_axis_point1 = '0 0 0'
+    cylindrical_axis_point2 = '0 1 0'
+  []
+[]
+
+[Constraints]
+  [mid_section_plane]
+    type = EqualValueBoundaryConstraint
+    variable = disp_y
+    secondary = top # boundary
+    penalty = 1.0e+10
+  []
+[]
+
+[Materials]
+  [swelling]
+    type = ADGenericFunctionMaterial
+    prop_values = swelling_func
+    prop_names = swelling
+  []
+  [elasticity_tensor]
+    type = ADComputeIsotropicElasticityTensor
+    youngs_modulus = 200.0e9
+    poissons_ratio = 0.2
+  []
+
+  [elastic_strain]
+    type = ADComputeMultipleInelasticStress
+    inelastic_models = "plasticity"
+    max_iterations = 50
+    absolute_tolerance = 1e-30 #1e-16
+  []
+
+  [hill_tensor]
+    type = ADHillConstants
+    # F G H L M N
+    # hill_constants = "0.5 0.5 0.5 1.5 1.5 1.5"
+    hill_constants = "0.5 0.25 0.5 1.5 1.5 1.5"
+  []
+
+  [plasticity]
+    type = ADHillElastoPlasticityStressUpdate
+    hardening_constant = 1.5e10
+    hardening_exponent = 1.0
+    yield_stress = 0.0 # 60e6
+    local_cylindrical_csys = true
+    # local_spherical_csys = false
+    axis = y
+    absolute_tolerance = 1e-15 # 1e-8
+    relative_tolerance = 1e-13 # 1e-15
+    internal_solve_full_iteration_history = true
+    max_inelastic_increment = 2.0e-6
+    internal_solve_output_on = on_error
+  []
+[]
+
+[BCs]
+  [no_disp_x]
+    type = ADDirichletBC
+    variable = disp_x
+    boundary = x_face
+    value = 0.0
+  []
+
+  [no_disp_y]
+    type = ADDirichletBC
+    variable = disp_y
+    boundary = bottom
+    value = 0.0
+  []
+
+  [no_disp_z]
+    type = ADDirichletBC
+    variable = disp_z
+    boundary = z_face
+    value = 0.0
+  []
+
+  [Pressure]
+    [Side1]
+      boundary = inner
+      function = push
+    []
+  []
+[]
+
+[Executioner]
+  type = Transient
+
+  solve_type = NEWTON
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = 'lu     superlu_dist'
+
+  nl_rel_tol = 1e-12
+  nl_abs_tol = 1e-14
+  # nl_abs_tol = 1e-10
+  l_max_its = 90
+  nl_max_its = 30
+  [TimeStepper]
+    type = IterationAdaptiveDT
+    optimal_iterations = 30
+    iteration_window = 9
+    growth_factor = 1.05
+    cutback_factor = 0.5
+    timestep_limiting_postprocessor = matl_ts_min
+    dt = 0.1e-4
+    time_t = '0 6.23 10'
+    time_dt = '0.1 1.0e-2 1.0e-2'
+  []
+  num_steps = 3
```

Comment:
Can you run one time step here to reduce the test runtime (up to 1200s)? Maybe you can set it up such that you get inelastic strain within one time step.

Reply:
Yes, changed to single step and plasticity starts right at the beginning.